docs/source/1.0/spec/aws/aws-query-protocol.rst

```diff
@@ -58,6 +58,81 @@ See
             }
         }
 
+.. _aws.protocols#awsQueryError-trait:
+
+-------------------------------------
+``aws.protocols#awsQueryError`` trait
+-------------------------------------
+
+Summary
+    Provides the value in the "Code" distinguishing field and HTTP response
+    code for an operation error. :ref:`See awsQuery error serialization. <awsQuery-errors>`
+Trait selector
+    ``structure [trait|error]``
+
+    The ``httpError`` trait can only be applied to :ref:`structure <structure>`
+    shapes that also have the :ref:`error-trait`.
+Value type
+    ``structure``
+
+.. important::
+    The ``aws.protocols#awsQueryError`` trait is only used when serializing
+    operation errors using the ``aws.protocols#query`` protocol.
+
+The ``awsQueryError`` trait is a structure that supports the following members:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 10 25 65
+
+    * - Property
+      - Type
+      - Description
+    * - code
+      - ``string``
+      - **Required** The value used to distinguish this error shape during
```

Comment:
It's really used for clients when deserializing errors. Might be worth rephrasing to make that clear. I know the service serializes this, but the only actual point of "Code" is for clients to find the right type.